client/src/components/title/Title.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import classnames from "classnames";
+
+import "./title.css";
+
+type TitleProps = {
+    text: string;
+    className?: string;
+    level: 1 | 2 | 3 | 4 | 5 | 6;
+    dark: boolean;
+};
+
+export const Title = (props: TitleProps) => {
+    const { text, className, level, dark } = props;
```

Comment:
Можно деструктурировать сразу в аргументе функции

```suggestion
export const Title = ({ text, className, level, dark }: TitleProps) => {
```

Reply:
Изначально хотел так и сделать, но сделал по аналогии с готовым компонентом button, чтобы все в едином стиле было)